FAQs: [CDS-98339]: FAQs Addition for 21st June (#7165)
harness/developer-hub: docs/continuous-delivery/deploy-srv-diff-platforms/aws/aws-deployment-faqs.md

### Does Harness support viewing the rendered launch template for ASG deployments after all the Harness expressions have been evaluated?

No, Harness does not store the rendered launch template as an output variable, nor does it display the rendered launch templates in the console logs. We only indicate that the ASG configuration is created. For more details, go to [Harness ASG services](https://developer.harness.io/docs/continuous-delivery/deploy-srv-diff-platforms/aws/asg-tutorial/#harness-asg-services).

### Does Harness support using IRSA with ECS for deployment?

> No, IRSA is not used with ECS. Instead, tasks in ECS get their own task roles, which can be set up similarly to IRSA by having a delegate in ECS run with a base role and using STS to assume a secondary role.

Reviewer: ```suggestion
No, IRSA is not used with ECS. Instead, ECS tasks get their own task roles, which is set up similar to IRSA. A delegate runs in ECS with a base role, STS is used to assume a secondary role.
```

> ### Does Harness support using serverless CLI on a delegate?

Reviewer: ```suggestion
### Does Harness support using Serverless CLI on a delegate?
```

> Yes, on using the serverless CLI on the delegate, ensure the delegate has the necessary permissions and environment setup. This includes adding node/serverless/aws CLI to the Delegate INIT script and managing sts

Reviewer: ```suggestion
Yes, using the Serverless CLI on the delegate ensures that the delegate has the necessary permissions and environment setup. This includes adding node, Serverless, or AWS CLI to the delegate INIT script and managing STS.
```

> Yes, on using the serverless CLI on the delegate, ensure the delegate has the necessary permissions and environment setup. This includes adding node/serverless/aws CLI to the Delegate INIT script and managing sts
> /trust configurations appropriately for each environment and service role.

Reviewer: ```suggestion
Yes, using the Serverless CLI on the delegate ensures that the delegate has the necessary permissions and environment setup. This includes adding node, Serverless, or AWS CLI to the delegate INIT script and managing STS/trust configurations appropriately for each environment and service role.
```

### Does Harness support conditional input variables based on previous selections?

No, currently Harness does not support dynamically showing different input variables based on previous selections during pipeline execution.

> Please read more on this in [Stage and step conditional execution settings](/docs/continuous-delivery/x-platform-cd-features/executions/step-and-stage-conditional-execution-settings/#and-execute-this-stage-only-if-the-following-jexl-condition-evaluates-to-true)

Reviewer: ```suggestion
For more details, go to [Stage and step conditional execution settings](/docs/continuous-delivery/x-platform-cd-features/executions/step-and-stage-conditional-execution-settings/#and-execute-this-stage-only-if-the-following-jexl-condition-evaluates-to-true).
```

### Does Harness support displaying inputs only related to selected values in a pipeline?

> No, Harness does not support conditional display of input variables based on previous selections, similar to how selecting an environment displays associated infrastructures.

Reviewer: ```suggestion
No, Harness does not support conditional displaying input variables based on previous selections.
```

### Does Harness support using allowed values for dynamic input options?

Yes, you can use allowed values to specify different input options for a variable, though it does not fully cover dynamic input based on previous selections.

> ### 

Reviewer: ```suggestion

```
